main/EDPftvarcon.F90

```diff
@@ -1858,33 +1856,6 @@ subroutine FatesCheckParams(is_master)
 
      if(.not.is_master) return
 
```

Comment:
Noting that we have xml level guardrails on valid namelist inputs, so we don't need to add these checks somewhere else.
https://github.com/ESCOMP/CTSM/pull/2904/files#diff-3e692a9856bdea4592f7a24859b6967201505af3780e800abe3d906c8b7ef7adR778